configs/components/oifs/oifs.yaml

```
@@ -1340,26 +1348,15 @@ oifs:
     #======================================================================================
 
     choose_output:
-            "default":
-                    add_namelist_changes:
-                            fort.4:
-                                    NAMCT0:
-                                            NFRPOS: "$(( 3600 / ${time_step} * 6 ))"
-                                            NFRHIS: "$(( 3600 / ${time_step} * 6 ))"
-
+            "default": #6hr
```

What about getting rid of the `default` as an option, and setting `output: "6hr"` as a default above the `choose_output`? I think that makes more sense, because then all options of the output follow the same logic.

```suggestion
            "6hr":
```